shared_model/interfaces/transaction.hpp

```diff
@@ -103,7 +96,6 @@ namespace shared_model {
             .append("hash", hash().hex())
             .append("txCounter", std::to_string(transactionCounter()))
             .append("creatorAccountId", creatorAccountId())
-            .append("quorum", std::to_string(quorum()))
```

Comment:
Why you removed it?

Reply:
https://github.com/hyperledger/iroha/pull/725#discussion_r153432324